web/styles/rendered_markdown.css

```diff
@@ -428,6 +428,40 @@
         float: none;
     }
 
+    .message_inline_video {
+        position: relative;
+        margin-bottom: 5px;
+        height: 150px;
+        display: inline-block;
+
+        &:hover {
+            &:not(.play_button_hidden)::after {
+                transform: scale(1.5);
+            }
+        }
+
+        &:not(.play_button_hidden)::after {
+            content: "";
+            background-image: url("../images/play_button.svg");
+            display: block;
+            width: 24px;
+            height: 24px;
+            position: absolute;
+            top: 63px;
+            left: 113px;
+            border-radius: 100%;
+            transition: transform 0.2s;
+        }
+
+        > video {
+            position: relative;
+            height: 150px;
+            width: 250px;
+            border-radius: 4px;
+            object-fit: cover;
```

Comment:
I don't think `object-fit: cover` is the right choice here (`contain` would be the better option, I think). `cover` looks nice and uniform, but it's going to cause parts of the video to be cut off that aren't in the (unusual) 5:3 aspect ratio set up by the height and width here. A 16:9 ratio (standard for widescreen HD) would call for opening the width up to 267px; users uploading a 4:3 video would see letterboxing on the sides familiar from 16:9 TVs.

Reply:
Thanks, changed to use contain!